src/herder/HerderImpl.cpp

```diff
@@ -1008,7 +1073,7 @@ HerderImpl::triggerNextLedger(uint32_t ledgerSeqToTrigger,
 
```

Comment:
Are we using `slotIndex` here correctly in the case when we're closing with a distant `ledgerSeq`? (Wasn't allowed to comment on the line I wanted, but it's right above this)

Reply:
Hmm, I'm not sure.  From my first reading I thought that when we're in `RUN_STANDALONE` mode, it sufficed for `slotIndex` to be increasing, but maybe it does have to match the new ledger sequence number.

I've changed it to be the latter because that now looks more plausible to me, but I'm still not sure what are all the ways that `slotIndex` is used.  This is probably a question for @MonsieurNicolas : if an enhanced manual ledger close increases the sequence number by more than `1`, should the `slotIndex` be one greater than the old sequence number, or equal to the new, even greater sequence number?  Or does it require subtler handling than either of those extremes?

Reply:
I'm hoping that this has been addressed in the latest implementation, which reverts to using `triggerNextLedger()` only to increment the sequence number by `1` (it currently uses `advanceLedgerPointers()` to do the big jump).